nservicebus/upgrades/support-policy.md

```diff
@@ -9,7 +9,7 @@ related:
 
 ## Supported Versions
 
-Supported versions include any major version released within the last 3 years. Within a supported major version, the latest minor version is supported, or any minor version released within the last year.
+Supported versions include any major version released within the last 3 years. Within a supported major version, the latest minor version is supported. For the NServiceBus package only (i.e. excluding downstream packages such as [transports](/nservicebus/transports/) and [persistence](/nservicebus/persistence/)), any minor version released within the last year is supported.
```

Comment:
@adamralph - I am not sure what is being conveyed here. Are you implying this policy applies only for NServiceBus package and not for other downstream packages such as transports, persistence and containers? What about the minor downstream packages then?